test/functional/specs/Personalization/C1234567.js

```diff
@@ -345,3 +345,359 @@ test("Test C1234567: Subscribes content cards", async () => {
   await responseStatus(edgeEndpointLogs.requests, [200, 204]);
   await t.expect(edgeEndpointLogs.count(() => true)).eql(2);
 });
+
+test("Test C1234567: Content card expiration", async () => {
+  const surface = "web://mywebsite.com/#expired-cards";
+
+  const mockPublishedDate = Math.ceil(new Date().getTime() / 1000) - 864000;
+  const mockExpiryDate = Math.ceil(new Date().getTime() / 1000) - 3600; // 1 hour ago
+
+  const activityId = uuid();
+  const propositionId = uuid();
+  const itemId = uuid();
+
+  const responseBody = createMockResponse({
```

Comment:
@shammowla  Now that Content Card is released in the UI, why are we still mocking the response?